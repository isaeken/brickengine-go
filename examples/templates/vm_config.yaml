machine:
  name: "vm-{{ slug(vars.hostname) | random_string(5) }}"
  ip: "{{ random_ipv4_in('10.0.0.0/24') }}"
  mac: "{{ random_mac() }}"
