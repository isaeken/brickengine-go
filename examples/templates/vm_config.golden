machine:
  name: "vm-random-generated-string"
  ip: "random-generated-ip4"
  mac: "random-generated-mac-address"
